Makefile

```diff
@@ -33,8 +32,7 @@ lint:
 
 .PHONY: gen-proto
 gen-proto:
-	${GOBIN}/buf generate --template buf.gen.gocosmos.yaml --exclude-path $(PULSAR_PATHS)
-	${GOBIN}/buf generate --template buf.gen.pulsar.yaml --path $(PULSAR_PATHS)
+	${GOBIN}/buf generate --template buf.gen.gocosmos.yaml
```

Comment:
If memory serves me right, I think we can `buf.gen.gocosmos.yaml` to `buf.gen.yaml` and simplify this to `${GOBIN}/buf generate`. 

https://github.com/cosmos/cosmos-sdk/issues/20410#issuecomment-2115270916

Reply:
Good call, I'll update that! I'll probably leave this as three separate commits for merging, one for each dependency upgrade and one for removing the pulsar files, updating the Makefile, and generating the new module configs with gocosmos.

Reply:
> one for removing the pulsar files, updating the Makefile, and generating the new module configs with gocosmos.

Oh yeah, good idea to keep these changes together. Theoretically, the CI checks should pass on every individual commit merged to main.

Reply:
Just realized that I didn't update the Makefile to remove the `buf.gen.yaml` template, I'll include that in the next refactor PR